bookkeeper-server/src/test/java/org/apache/bookkeeper/conf/AbstractConfigurationTest.java

```diff
@@ -56,6 +56,12 @@ public void setup() {
         this.conf.setZkLedgersRootPath("/path/to/ledgers");
     }
 
+    @Test
+    public void testSetGetServiceUri() throws Exception {
+        this.conf.setMetadataServiceUri("zk1:2181,zk2:2181/test");
```

Comment:
this is not a valid URI

it should be something like `zk://zk1:2181,zk2:2181/test` or `zk+null://zk1:2181,zk2:2181/test`

Reply:
https://github.com/apache/pulsar/blob/d02bd7378acd1f202d8bcf50aa6999749e61b20b/pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java#L246-L248